### DISCUSSION TITLE
CTk Components

### DISCUSSION THREAD

**Discussion by rudymohammadbali** [TomSchimansky/CustomTkinter#2276] [Showcase]
<h1 align="center"><a href="https://github.com/rudymohammadbali/ctk_components">CTk Components</a></h1>
<p align="center">Version 20240226</p>
CTk Components is a collection of widgets and utilities designed as extensions or add-ons for customtkinter

## Widgets

1. `CTkAlert`
2. `CTkBanner`
3. `CTkNotification`
4. `CTkCard`
5. `CTkCarousel`
6. `CTkInput`
7. `CTkLoader`
8. `CTkPopupMenu`
9. `CTkProgressPopup`
10. `CTkTreeview`
## Utilities

The pack also includes 4 utilities:

- `window_position.py`
- `ctkgif.py`
- `py_win_style.py`
- `blue.json`

## Preview

https://github.com/rudymohammadbali/ctk_components/assets/63475761/ed48154a-9225-4c72-8264-9d42aef83cfb

## Example usage
Check [Wiki Page](https://github.com/rudymohammadbali/ctk_components/wiki) for example usage and arguments

## Support

<p align="left">If you'd like to support my ongoing efforts in sharing fantastic open-source projects, you can contribute by making a donation via PayPal.</p>

<div align="center">
  <a href="https://www.paypal.com/paypalme/iamironman0" target="_blank">
    <img src="https://img.shields.io/static/v1?message=PayPal&logo=paypal&label=&color=00457C&logoColor=white&labelColor=&style=flat" height="40" alt="paypal logo"  />
  </a>
</div>

**Comment by BhagyaJyoti22006** [comment]
kudos mate. These are a great addition to already existing widgets. Btw, under wiki page, can you also add 2-3 lines describing the widget. For example, what purpose does banner serves? I am just unable to recall its sightings.

**Reply by rudymohammadbali** [reply]
Thank you for your feedback!
